mycpp/gc_dict.h

```diff
@@ -155,9 +144,7 @@ class Dict {
   int capacity_;  // number of entries before resizing
 
   // These 3 slabs are resized at the same time.
-  Slab<int>* entry_;  // NOW: kEmptyEntry, kDeletedEntry, or 0.
-                      // LATER: indices which are themselves indexed by // hash
-                      // value % capacity_
+  Slab<int>* entry_;  // kEmptyEntry, or a valid index into keys_/values_
```

Comment:
Since we have a method name `find_key_in_index()`, should this be called `index_` ?  I think that's a better name anyway, not sure why I called it `entry_`

Reply:
Ah yeah. I wanted to do this rename, but it collides with the `index_()` function :( I guess we could have mycpp generate a different function call for `[]` expressions. Not sure if worth... Maybe `indices_`?

Reply:
Oh this is weird, I called that the `a[0]` function `a->index_(0)` because Python already has a different `a.index()` function

I think it's fine to leave it as is now, but I might rename things later

It could be `a.at(0)`, leaving room for the `a->index_` member

Tuples already have `t.at0()`, so this would make a lot of sense

Reply:
Oh, yes! `at()` is also consistent with how the STL names this function (e.g. `std::map.at()` throws an exception if the key isn't found)

Reply:
this rename actually ended up being pretty easy. i'm gonna add it...